Jellyfin.Server/Program.cs

```diff
@@ -139,7 +140,14 @@ private static async Task StartServer(IServerApplicationPaths appPaths, StartupO
                 host = Host.CreateDefaultBuilder()
                     .UseConsoleLifetime()
                     .ConfigureServices(services => appHost.Init(services))
-                    .ConfigureWebHostDefaults(webHostBuilder => webHostBuilder.ConfigureWebHostBuilder(appHost, startupConfig, appPaths, _logger))
+                    .ConfigureWebHostDefaults(webHostBuilder =>
+                    {
+                        webHostBuilder.ConfigureWebHostBuilder(appHost, startupConfig, appPaths, _logger);
+                        if (bool.TryParse(Environment.GetEnvironmentVariable("JELLYFIN_ENABLE_IIS"), out var iisEnabled) && iisEnabled)
+                        {
```

Comment:
```suggestion
                        {
                        _logger.LogCritical("UNSUPPORTED HOSTING ENVIORMENT Microsoft Internet Information Services. The option to run Jellyfin within an IIS is an unsupported and untested feature. Only use at your own digression.");
```

Plus a very clear and very explicit description in the docs and i am ok with it.

Reply:
I take it English is not your first language :) Should read more like

`UNSUPPORTED HOSTING ENVIRONMENT Microsoft Internet Information Services. The option to run Jellyfin within IIS is an unsupported and untested feature. Only use at your own digression.`

Reply:
After some deliberation with some other team members, it was decided that "on" would be better

Reply:
`ENVIORMENT` should be `ENVIRONMENT` :)

Reply:
You know you can just push the change yourself right?

Reply:
friendly reminder that this space is for code-focused, results-oriented discussion.

Reply:
I'm headed to work, lol.... I could later

Reply:
@kimboslice99 i now feel like need to warn you specifically about maintaining a professional tone. My interpretation of this thread is that you are not showing Venson a reasonable level of respect here, and I encourage you to address that shortcoming so that we can all maintain a professional relationship.

Reply:
one way that you could submit this exact same feedback without coming off as flippant or condescending is to actually push the required changes in. that makes it pretty incontrovertable that you have good intentions -- something that is missing here.

Reply:
@sevenrats Did not see your comment. 

Not sure how we've gotten so off track here. I simply did not have time, I thought perhaps if he felt like it he could correct it while I was preoccupied.

Anyways, I will accept the change. Thanks for your input.